evaluator/autoware_planning_evaluator/launch/planning_evaluator.launch.xml

```diff
@@ -8,11 +9,16 @@
   <arg name="input/map_topic" default="/map/vector_map"/>
   <arg name="input/route_topic" default="/planning/mission_planning/route"/>
   <arg name="input/diagnostics" default="/diagnostics"/>
+  <!-- log -->
+  <arg name="result_archive_path" default=""/>
+  <let name="output_file" value="" if="$(eval &quot;'$(var result_archive_path)' == ''&quot;)"/>
+  <let name="output_file" value="$(var result_archive_path)/planning_evaluator/metric.log" unless="$(eval &quot;'$(var result_archive_path)' == ''&quot;)"/>
```

Comment:
In the local environment when we run Autoware, is the planning_evaluator's log saved? If yes, where will it be?

Reply:
before this PR, the `output_file` seems never be used despite its existence in the [planning_evaluator](https://github.com/autowarefoundation/autoware.universe/blob/00a9a343376448a47d1d61c7d667b92a3b2cf43b/evaluator/autoware_planning_evaluator/src/planning_evaluator_node.cpp#L58).

I'm planning to let other evaluators write the respective logs to `result_archive_path` with a similar way(output_file = $result_archive_path/node_name/something.log). Is it a proper evaluation design?

Reply:
I wanna know the detail global path of the following in the local environment.
(output_file = ${result_archive_path}/node_name/something.log)